x-pack/plugins/apm/public/components/app/ServiceOverview/ServiceList/index.js

```diff
@@ -7,19 +7,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import numeral from '@elastic/numeral';
 import { RelativeLink } from '../../../../utils/url';
 import { fontSizes, truncate } from '../../../../style/variables';
 import TooltipOverlay from '../../../shared/TooltipOverlay';
-import { asMillisWithDefault } from '../../../../utils/formatters';
+import { asMillisWithDefault, asInteger } from '../../../../utils/formatters';
 import { ManagedTable } from '../../../shared/ManagedTable';
 
 // TODO: Consolidate these formatting helpers centrally
 function formatNumber(value) {
   if (value === 0) {
     return '0';
   }
-  const formatted = numeral(value).format('0.0');
+  const formatted = asInteger(value);
```

Comment:
Actually - should this perhaps be `asDecimal` instead? We used to have decimals (I noticed the changed snapshots).

Reply:
Ah ok, I wasn't sure if your suggestion to use `asInteger` was on purpose to switch this to an integer or not. Will update.